### ISSUE TITLE
Tests fail after installation

### ISSUE THREAD

**Issue by PepIngla** [enthought/mayavi#835]
Hello, 
I have installed mayavi on anaconda and, when I run the tests using: nosetests mayavi/tests
I get the following error repeated several times (I can read 7):
======================================================================
ERROR: Failure: SyntaxError (invalid syntax (state_pickler.py, line 1210))
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\jingla\AppData\Local\Continuum\anaconda3\lib\site-packages\nose
\failure.py", line 39, in runTest
    raise self.exc_val.with_traceback(self.tb)
  File "C:\Users\jingla\AppData\Local\Continuum\anaconda3\lib\site-packages\nose
\loader.py", line 417, in loadTestsFromName
    addr.filename, addr.module)
  File "C:\Users\jingla\AppData\Local\Continuum\anaconda3\lib\site-packages\nose
\importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "C:\Users\jingla\AppData\Local\Continuum\anaconda3\lib\site-packages\nose
\importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "C:\Users\jingla\AppData\Local\Continuum\anaconda3\lib\imp.py", line 234,
 in load_module
    return load_source(name, filename, file)
  File "C:\Users\jingla\AppData\Local\Continuum\anaconda3\lib\imp.py", line 171,
 in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 696, in _load
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\jingla\AppData\Local\Continuum\anaconda3\pkgs\mayavi-4.6.2-py37
h3ccff09_4\Lib\site-packages\mayavi\tests\test_vtk_file_reader.py", line 11, in
<module>
    from mayavi.sources.vtk_file_reader import VTKFileReader
  File "C:\Users\jingla\AppData\Local\Continuum\anaconda3\pkgs\mayavi-4.6.2-py37
h3ccff09_4\Lib\site-packages\mayavi\sources\vtk_file_reader.py", line 19, in <mo
dule>
    from .vtk_xml_file_reader import VTKXMLFileReader
  File "C:\Users\jingla\AppData\Local\Continuum\anaconda3\pkgs\mayavi-4.6.2-py37
h3ccff09_4\Lib\site-packages\mayavi\sources\vtk_xml_file_reader.py", line 16, in
 <module>
    from mayavi.core.common import error
  File "C:\Users\jingla\AppData\Local\Continuum\anaconda3\pkgs\mayavi-4.6.2-py37
h3ccff09_4\Lib\site-packages\mayavi\core\common.py", line 16, in <module>
    from apptools.persistence.state_pickler import create_instance
  File "C:\Users\jingla\AppData\Local\Continuum\anaconda3\lib\site-packages\appt
ools\persistence\state_pickler.py", line 1210

    ^
SyntaxError: invalid syntax
can someone help me with this?
cheers,
Josep

**Comment by prabhuramachandran** [comment]
See this: https://github.com/enthought/apptools/issues/79
Try this for a quick fix:
```
$ pip uninstall apptools
$ pip install https://github.com/enthought/apptools/zipball/master
```

**Comment by PepIngla** [comment]
Thank you for your reply, 
I am using python3.7.3 for anaconda and conda as a package manager.
To install the package, I downloaded it and unzipped it in a suitable folder. Then, from that folder I used: 
$conda install traitsui
$conda install configobject
$python setup.py install
$conda install -c conda-forge mayavi
 and it worked. The first two lines I used because otherwise I was getting errors.
Using 
$conda install mayavi 
didn't work (I think that it is because it installs mayavi 4.6.2 using apptools 4.4.0). 
Updating the package using 
$conda update mayavi 
leads to the exact same result reproducing the error described above.
After this I removed mayavi and installed it again using 
$conda install -c conda-forge mayavi
and it worked again. I couldn't remove "apptools 4.6.0.dev0" using
$conda remove apptools 
to test if this was required or apptools 4.5.0 is enough.
